ts-perf/packages/api/src/options.ts

```diff
@@ -43,21 +43,17 @@ const suiteDir: CommandLineOption = {
         "Use <directory> as the root location for test suites (i.e. './cases/solutions'). If not set, uses TSPERF_SUITE_DIR environment variable, if found. Otherwise, uses '~/.tsperf/solutions', if present.",
 };
 
-const scenarioDirs: CommandLineOption = {
+const scenarioDir: CommandLineOption = {
     type: "string",
     longName: "scenarioDir",
     alias: ["scenarioDirs", "scenarioConfigDir", "scenarioConfigDirs"],
-    multiple: true,
     validate: validatePath,
     defaultValue() {
-        const dirs = process.env.TSPERF_SCENARIO_DIRS?.split(path.delimiter)
-            .map(dirname => findPath(dirname, /*relative*/ undefined, /*walkUpParents*/ false))
-            .filter((dirname): dirname is string => !!dirname);
-        return dirs;
+        return findPath(process.env.TSPERF_SCENARIO_DIR, /*relative*/ undefined, /*walkUpParents*/ false);
```

Comment:
Just to be clear, were you just making this change due to my comments in the previous PR, or is this just generally better anyway? I wasn't suggesting we remove it if it's useful, but was just commenting on the one `add` command that switched to a single dir. Which in retrospect, I misunderstood, as it of course makes sense for `add` to only apply to one directory...

Reply:
I'm removing it because it makes sense that there's only ever two scenario directories, max: the default one created in `~/.tsperf/scenarios` and a manually configured one by way of `--scenarioDir` or `%TSPERF_SCENARIO_DIR%`. Supporting _n_ root-level scenario directories seems unnecessary and more trouble than it's worth.